src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Helpers.cs

```diff
@@ -13,6 +13,21 @@ public static partial class JsonSerializer
         internal const string SerializationUnreferencedCodeMessage = "JSON serialization and deserialization might require types that cannot be statically analyzed. Use the overload that takes a JsonTypeInfo or JsonSerializerContext, or make sure all of the required types are preserved.";
         internal const string SerializationRequiresDynamicCodeMessage = "JSON serialization and deserialization might require types that cannot be statically analyzed and might need runtime code generation. Use System.Text.Json source generation for native AOT applications.";
 
+        /// <summary>
+        /// Indicates whether unconfigured <see cref="JsonSerializerOptions"/> instances
+        /// should be set to use the reflection-based <see cref="DefaultJsonTypeInfoResolver"/>.
+        /// </summary>
+        /// <remarks>
+        /// The value of the property is backed by the "System.Text.Json.JsonSerializer.IsReflectionEnabledByDefault"
+        /// feature switch and defaults to <see langword="true"/> if unset. For trimmed applications, disabling the feature switch
+        /// at link time will result in the property being substituted with a constant <see langword="false"/> value.
```

Comment:
```suggestion
        /// will result in the property being substituted with a constant <see langword="false"/> value.
```

Reply:
Why do you think it should be removed? AppContext switches can also be set at run time from within the application.

Reply:
1. "link time" isn't really a concept in .NET. If you really need to specify a time here, "publish time" would be more appropriate.
2. Feature switches work both with and without actually trimming. It might be misleading to consumers. They might think the value only turns to `false` when I actually publish/trim my app.

Maybe this 2nd sentence should just be removed?

Reply:
Sure I can just remove it.